node/src/Router.ts

```diff
@@ -644,6 +645,14 @@ export class Router<RouterAppData extends AppData = AppData>
 			throw new TypeError('if given, appData must be an object');
 		}
 
+		// If rtcpMux is enabled, ignore rtcpListenInfo.
+		if (rtcpMux && rtcpListenInfo !== undefined)
```

Comment:
We can unset `rtcpListenInfo` unconditionally.

```suggestion
		if (rtcpMux)
```